packages/server/test/integration/http_requests_spec.coffee

```diff
@@ -262,14 +262,14 @@ describe "Routes", ->
           expect(res.statusCode).to.eq(200)
           expect(res.body).to.match(/Runner.start\(.+\)/)
 
-    it "routes even without a proxy set", ->
+    it "routes to 'not launched through Cypress' without a proxy set", ->
```

Comment:
I think you need to add **another** test that hits one of the other routes that's locked down behind a proxy to ensure that it redirects you to `/__`.

I don't think there's any logic testing that you get a redirect for other locked down / secured routes.